openedx/features/caliper_tracking/tests/tests.py

```diff
@@ -25,7 +26,12 @@ class CaliperTransformationTestCase(TestCase):
 
     maxDiff = None
 
-    def test_caliper_transformers(self):
+    @mock.patch(
+        'openedx.features.caliper_tracking.utils.get_username_from_user_id',
+        return_value='edx',
```

Comment:
`edx` is also the name of organizaion. Can you please use a more appropriate name?

Reply:
done.